xml/installation-installation-ses_integration.xml

```diff
@@ -9,62 +9,33 @@
  <title>&ses; Integration</title>
  <para>
   The current version of &productname; supports integration with &ses;
-  (SES). Integrating &ses; enables &ceph; block storage as well as object and
-  image storage services in &productname;.
+  (SES). Integrating &ses; enables &ceph; to provide RADOS Block Device (RBD),
+  block storage, image storage, object storage via RADOS Gateway (RGW),
+  and CephFS (file storage) in &productname;. The following documentation
+  outlines integration for &ses; 5 and 5.5.
  </para>
+ <para>
+   &ses; 5.5 utilises a Salt runner that creates users and pools. Salt generates
+   a yaml configuration that is needed to integrate with &cloud;.
+   The integration runner creates separate users for &o_blockstore;,
+   &o_blockstore; backup, and &o_img;. Both the &o_blockstore;
+   and &o_comp; services have the same user, as &o_blockstore; needs access
+   to create objects that &o_comp; uses.
+ </para>
+ <para>
+   &ses; 5 is a manual configuration that requires the creation of users and
```

Comment:
Seems like this needs a little something different. SES 5 uses/utilizes a manual configuration...